### PR TITLE
9.0 cresus import bis

### PR DESCRIPTION
**Pull request by ghost**
These are just some simple fixes to l10n_ch_import_cresus, which I've previously ported to 9.0 (#253) before touching another similar module, l10n_ch_import_winbiz, which is still in PR (#255). Working again on the same kind of things made me learn some things I didn't know the first time, so I had to backport a bit. I also took the opportunity to fix some minor details in the README.rst.

### CODE REVIEW

File: l10n_ch_import_cresus/tests/test_import.py [hunk 1]
```diff
@@ -42,32 +47,63 @@ def setUp(self):
...
+        if not misc:
+            misc = journal_obj.create({
+                'code': 'MISC',
+                'name': 'dummy MISC',
+                'type': 'general'})
+
+        def get_path(filename):
+            res = get_resource_path('l10n_ch_import_cresus', 'tests', filename)
+            return res
+        input = open(get_path('input.csv'))
```
Comment (yvaucher) [inline l10n_ch_import_cresus/tests/test_import.py]: Please don't use built-in `input` function name for your variable.

Let's say you remove this line later, you won't get an error of undefined variable.

**Comment by yvaucher** [comment]
Just a tiny remark otherwise looks good to me. :+1:

File: l10n_ch_import_cresus/tests/test_import.py (hunk 1 above)
Comment (ghost) [inline l10n_ch_import_cresus/tests/test_import.py]: I see. Also its scope is way bigger than it needs be. Gotta fix that.

File: l10n_ch_import_cresus/tests/test_import.py [hunk 2]
```diff
@@ -42,32 +47,64 @@ def setUp(self):
...
-        self.assertEqual(res[2].date, '2002-01-07')
-        self.assertEqual(res[3].line_ids[0].account_id.code, '6642')
-        self.assertEqual(res[3].line_ids[0].debit, 15.0)
-        self.assertEqual(res[3].line_ids[0].credit, 0.0)
-        self.assertEqual(res[3].line_ids[1].account_id.code, '1010')
-        self.assertEqual(res[3].line_ids[1].debit, 0.0)
-        self.assertEqual(res[3].line_ids[1].credit, 15.0)
-        self.assertEqual(res[4].line_ids[1].tax_line_id, self.vat)
+
+        gold = open(get_path('golden-output.txt'))
```
Comment (yvaucher) [inline l10n_ch_import_cresus/tests/test_import.py:75]: This file is opened but not closed, you might want to use `with` syntax or call `close()`

File: l10n_ch_import_cresus/models/account_tax.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2015 Camptocamp SA
+# Copyright 2016 Open Net Sàrl
```
Comment (yvaucher) [inline l10n_ch_import_cresus/models/account_tax.py]: You can add your copyright but you cannot remove existing one, it is AGPL infrigement event when you provide those modules to your customer, you have to respect the AGPL license.

File: l10n_ch_import_cresus/wizard/l10n_ch_import_cresus.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2015 Camptocamp SA
+# Copyright 2016 Open Net Sàrl
```
Comment (yvaucher) [inline l10n_ch_import_cresus/wizard/l10n_ch_import_cresus.py]: This is copyright stealing you can not do that.

First version of the module is there:
https://github.com/OCA/l10n-switzerland/commit/09c366359b157afb3b85a0684b7cc23d603624ef

**Review by yvaucher — CHANGES_REQUESTED** [review]
The change seems okay, `include_initial_balance` is not used anymore.

However, I'm really disapointed that the copyright is changed. OpenNET has been there for a while and should know the rules and what AGPL is.

In fact your changes of copyright does not respect the section 4 of AGPL. You have to keep original copyright notices whenever the copyright would apply without those mentions in most of the countries. Whenever you redistribute this code you do have to keep the right notices. It's not the first time I have seen such changes in your pull requests. You have to stop doing that !

```
4. Conveying Verbatim Copies.

You may convey verbatim copies of the Program's source code as you receive it, in any medium, provided that you conspicuously and appropriately publish on each copy an appropriate copyright notice; keep intact all notices stating that this License and any non-permissive terms added in accord with section 7 apply to the code; keep intact all notices of the absence of any warranty; and give all recipients a copy of this License along with the Program.
```

**Comment by ghost** [comment]
I'm sorry

**Comment by yvaucher** [comment]
Almost.

This is done by 1 copyright per line in all OCA modules

Thus you should have:

```
# -*- coding: utf-8 -*-
# Copyright 2015 Camptocamp SA
# Copyright 2016 Open Net Sàrl
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

**Comment by ghost** [comment]
fixed, thank you very much

**Comment by ghost** [comment]
Is that also the case if the year is the same ?

```
# © 2016 Julien Coux (Camptocamp), Open Net Sàrl
```

or

```
# © 2016 Julien Coux (Camptocamp)
# © 2016 Open Net Sàrl
```

?

**Comment by yvaucher** [comment]
Prefer `Copyright` in full letters instead of © to be more explicit. (at some point the community choose it was better to be explicit)

Thus it is the same, better to have one line per author.

**Comment by yvaucher** [comment]
To be clear:

    # Copyright 2016 Julien Coux (Camptocamp)
    # Copyright 2016 Open Net Sàrl

Is the way to go

File: l10n_ch_import_cresus/tests/test_import.py (hunk 2 above)
Comment (ghost) [reply l10n_ch_import_cresus/tests/test_import.py:75]: fixed by 6a0891e45e0c15b6829f5d8c8399d65e6247c2fd

**Comment by ghost** [comment]
@yvaucher I request a new review.

Maybe a [rebase](https://github.com/open-net-sarl/l10n-switzerland/tree/9.0_cresus-import_rebase) is in order?  It should be strictly equivalent to [the current PR when merged](https://github.com/OCA/l10n-switzerland/tree/refs/pull/256/merge), while not polluting the commit graph. I also squashed fixup-type commits and deleted a commit-revert pair.

**Review by yvaucher — APPROVED** [review]

**Comment by yvaucher** [comment]
Okay we are good, yes please squash those last commit before I merge.

**Comment by ghost** [comment]
@yvaucher rebase done.

**Pull request merged**